pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -175,6 +175,14 @@ def test_to_datetime_format_weeks(self):
 
 class TestToDatetime(object):
 
+    def test_to_datetime_iso_week_year_format(self):
```

Comment:
https://github.com/pandas-dev/pandas/pull/16661#discussion_r121285015

I mean exactly this. you have an if clause so we need a test for both cases.

>  ordinal may be negative or 0 now, which means the date is in the previous calendar year